features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -1218,6 +1218,42 @@
             <Scopes>internal_identity_mgt_create</Scopes>
             <Scopes>internal_identity_mgt_delete</Scopes>
         </Resource>
+        <Resource context="(.*)/api/server/v1/email/template-types" secured="true" http-method="GET">
```

Comment:
(.*)/api/server/v1/email/template-types/(.*)

Reply:
using **(.)** is not applicable because there will be more than one character before **/api** . same for **/(.)**.
Recommended url for this api is **/email/template-types**, so using **template-types/** causes mismatch.